temperature/crud.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import select, insert
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from temperature import models
```

Comment:
https://realpython.com/absolute-vs-relative-python-imports/#styling-of-import-statements
Serg asked you not to use relative import, fix it please